packages/keyring-controller/src/KeyringController.ts

```diff
@@ -427,6 +427,17 @@ export class KeyringController extends BaseControllerV2<
     return this.#keyring.getAccounts();
   }
 
+  /**
+   * Get encryption public key.
+   *
+   * @param account - An account address.
+   * @throws If the `account` does not exist or does not support the `getEncryptionPublicKey` method
+   * @returns Promise resolving to encyption public key of the `account` if one exists.
+   */
+  async getEncryptionPublicKey(account: string): Promise<string> {
```

Comment:
We should also support an optional argument `opts?: Record<string, unknown>` argument to pass to `this.#keyring.getEncryptionPublicKey`

See: https://github.com/MetaMask/KeyringController/blob/main/src/KeyringController.ts#L467